test/test_samples_osw/base/results_output.csv

```diff
@@ -13,7 +13,7 @@ project_national-7.osw,20.0,443.8,5.5,99.7,585.2,24.4,283.8,683.2,0.0,0.0,198.6,
 project_national-8.osw,20.0,390.4,11.0,49.8,53.3,28.1,52.3,490.3,0.0,0.0,113.4,434.3,176.2,50.6,0.0,0.0,19.4,0.0,0.0,0.0,0.0,638.0,990.2,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,39.2,0.0,1220.0,1220.0,1220.0,0.0,0.0,0.0,0.0,7404.5,6383.1,0.0,true,24.2,18.6,0.0,0.0,69.4,0.0,0.0,0.0,79.9,4704.8,30.0,0.0,0.0,0.0,0.0,1364.0,25.3,21.8,40.0,3591.4,31.5,0.0,192.1,0.0,0.0,1166.3,1335.8,0.0,102.8,0.0
 project_national-18.osw,20.0,443.8,11.0,73.7,0.0,0.0,374.5,586.6,0.0,0.0,198.6,0.0,235.9,81.9,0.0,0.0,65.4,0.0,0.0,0.0,0.0,1144.5,1491.5,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,400.3,0.0,0.0,0.0,0.0,1138.0,1138.0,0.0,0.0,3.0,86.0,3064.8,3012.0,0.0,true,0.0,0.0,0.0,0.0,110.6,0.0,0.0,0.0,130.9,2762.0,30.0,0.0,0.0,0.0,0.0,0.0,10.5,10.3,40.0,4674.9,40.1,0.0,241.5,0.0,0.0,201.2,201.2,0.0,60.4,0.0
 project_national-22.osw,20.0,194.4,5.5,102.2,449.8,20.1,283.7,0.0,0.0,0.0,57.4,0.0,0.0,4.3,0.0,0.0,137.9,0.0,0.0,0.0,0.0,428.5,1203.2,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,1741.9,0.0,0.0,617.0,617.0,0.0,0.0,7261.0,0.0,0.0,1552.4,0.0,true,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1220.3,30.0,0.0,0.0,0.0,0.0,0.0,0.0,5.3,20.0,4879.7,16.7,0.0,0.0,0.0,0.0,148.1,148.1,74.1,26.7,0.0
-project_national-27.osw,20.0,540.8,5.5,50.3,644.5,28.6,409.5,2614.4,0.0,0.0,157.1,0.0,767.5,11.4,0.0,34.7,353.9,0.0,0.0,0.0,0.0,801.3,1888.2,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.4,0.0,1483.6,0.0,1690.0,1690.0,2266.0,0.0,0.0,0.0,0.0,10497.8,4746.3,0.0,true,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,3221.0,30.0,0.0,0.0,0.0,0.0,2533.5,35.8,16.2,30.0,9695.6,33.1,0.0,0.0,0.0,0.0,1485.1,2023.8,0.0,70.4,0.0
```

Comment:
@joseph-robertson why is the order changing?

Reply:
Because of parallelization. The next "Latest results" commit should show that it's been corrected.